.github/actions/test-dotnet/Invoke-SolutionTests.ps1

```diff
@@ -43,27 +43,21 @@ foreach ($test in $tests) {
         '--no-build',
         '--nologo',
         '--logger', 'trx;LogFileName=test-results.trx'
+        # This is for xUnit ITestOutputHelper, see https://xunit.net/docs/capturing-output.
+        '--logger', 'console;verbosity=detailed'
         '--verbosity', $Verbosity
```

Comment:
Doesn't that override or conflict with the `'--verbosity', $Verbosity` below?

Reply:
No, since this verbosity is only for the given logger (you can set multiple loggers, and each with their own verbosity).